composer.lock

```diff
@@ -4,8 +4,8 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
         "This file is @generated automatically"
     ],
-    "hash": "ebe771ebefde29ab37094e27b1a8383d",
```

Comment:
@slde-flash I guess yes, since the `hash` property hashes the full composer.json file and it has changed.